docs/configuration/secure.md

```diff
@@ -0,0 +1,614 @@
+* Opera 17
+* Safari 9
+* Android 5.0
+* Java 8
+
+Older browser clients will not be able to reach the ILIAS installation!
+
+#### Apache2
+*(Version: 2.4.34,  OpenSSL 1.1.1c)*
+
+Add the following line INSIDE the `` <VirtualHost></VirtualHost> `` block:
+```
+<VirtualHost *:443>
+    ...
+    ServerName %HOSTNAME%
+    ...
+    SSLEngine on
+    SSLCertificateFile       /path/to/signed_cert_plus_intermediates
+    SSLCertificateKeyFile   /path/to/private/key
+```
+
+Add the following line OUTSIDE  the `` <VirtualHost></VirtualHost> `` block:
+```
+SSLProtocol             all -SSLv3 -TLSv1 -TLSv1.1
+SSLCipherSuite          ECDHE-ECDSA-AES256-GCM-SHA384:ECDHE-RSA-AES256-GCM-SHA384:ECDHE-ECDSA-CHACHA20-POLY1305:ECDHE-RSA-CHACHA20-POLY1305:ECDHE-ECDSA-AES128-GCM-SHA256:ECDHE-RSA-AES128-GCM-SHA256:ECDHE-ECDSA-AES256-SHA384:ECDHE-RSA-AES256-SHA384:ECDHE-ECDSA-AES128-SHA256:ECDHE-RSA-AES128-SHA256
+SSLHonorCipherOrder     on
+SSLCompression          off
+SSLSessionTickets       off
+```
+
+
+#### NGINX
+*(Version: 1.14.0,  OpenSSL 1.1.1c)*
+
+Add the following line INSIDE the server block:
+
+```
+server {
+    listen 443 ssl http2;
+    listen [::]:443 ssl http2;
+    ...
+    server_name %HOSTNAME%;
+    ...
+    # certs sent to the client in SERVER HELLO are concatenated in ssl_certificate
+    ssl_certificate /path/to/signed_cert_plus_intermediates;
+    ssl_certificate_key /path/to/private_key;
+    ssl_session_timeout 5m;
+    ssl_session_cache shared:SSL:16m;
+    ssl_session_tickets off;
+
+    ssl_protocols TLSv1.2;
+    ssl_ciphers 'ECDHE-ECDSA-AES256-GCM-SHA384:ECDHE-RSA-AES256-GCM-SHA384:ECDHE-ECDSA-CHACHA20-POLY1305:ECDHE-RSA-CHACHA20-POLY1305:ECDHE-ECDSA-AES128-GCM-SHA256:ECDHE-RSA-AES128-GCM-SHA256:ECDHE-ECDSA-AES256-SHA384:ECDHE-RSA-AES256-SHA384:ECDHE-ECDSA-AES128-SHA256:ECDHE-RSA-AES128-SHA256';
+    ssl_prefer_server_ciphers on;
+
+    ssl_dhparam /etc/ssl/private/dhparam.pem;
+
+
+```
+
+``ssl_dhparam /etc/ssl/private/dhparam.pem;``:
+
+This specifies a file with DH parameters for EDH (Ephemeral Diffie-Hellman) ciphers.  
+By default, NGINX will use the default DHE paramaters provided by openssl. This uses a weak key that gets lower scores.  
+Run ``openssl dhparam -out /etc/ssl/private/dhparam.pem 4096`` in terminal to generate it.
+
+We RECOMMEND to use the [Mozilla SSL Configuration Generator](https://ssl-config.mozilla.org/) to generate a suitable configuration and the [Qualys SSL Labs Tests](https://www.ssllabs.com/ssltest/) or the [High-Tech Bridge SSL Server Test](https://www.htbridge.com/ssl/) to check your settings. It is recommended, to reach a "A" rating as minimum.
+
+It is necessary to often revise these configuration.
+In best case, you allways use the latest "Modern" configuration.
+
+### Serve security related Headers
+
+To improve the security of your ILIAS users you SHOULD set the following Headers:
+(this is experience based configuration set of headers, may this can differ to your configuration/ usage scenario)
```

Comment:
Little bit of English
```suggestion
(this is an experience based configuration set of headers, this may differ from your configuration/usage scenario)
```